release_docs/RELEASE.txt

```diff
@@ -47,6 +47,16 @@ New Features
 
     Configuration:
     -------------
+    - Enabled instrumentation of the library by default in CMake for parallel
+      debug builds
```

Comment:
Not really sure whether this fits more under "new features" or "bugs"

Reply:
I think this is good.